sheets-import/README.md

```diff
@@ -1,102 +1,270 @@
-* **Server.gs** - This file contains server-side code that responds to user
-  interactions in the sidebar UI. It also sets up the add-on menu.
-* **Sidebar.html** - This file contains the HTML structure for that defines
-  the sidebar UI.
-* **Stylesheet.html** - This file contains all the CSS properties defined for
-  the template.
-* **Utilities.gs** - This file contains some generic functionalities to support
-  the rest of the code. The functions here are not specific to this template and
-  could be taken for use in other projects without modification.
-* **intercom.js.html** - This file contains a copy of
-  [intercom.js](https://github.com/diy/intercom.js),
-  a cross-window message broadcast interface (intercom.js is released under an
-  Apache V2.0 license).
-
-## Setup: Libraries
-This template makes use of the following libraries, which much the added to the
-Apps Script project before the template can be used:
-
-* [Apps Script OAuth2 library](https://github.com/googlesamples/apps-script-oauth2)
-* [Underscore](http://underscorejs.org/)
-
-These libraries are already published as an Apps Script, making it easy to
-include in your project. To add it to your script, do the following in the
-Apps Script code editor:
-
-1. Click on the menu item "Resources > Libraries..."
-1. In the "Find a Library" text box, enter the project key
-"MswhXl8fVhTFUH_Q3UOJbXvxhMjh3Sh48" and click the "Select" button.
-1. Choose the latest version in the dropdown box.
-1. Click the "Save" button.
-
-This will add the [OAuth2 library](https://github.com/googlesamples/apps-script-oauth2)
-to your project. Repeat the above steps with the project key
-"MGwgKN2Th03tJ5OdmlzB8KPxhMjh3Sh48" to add Underscore to the project as well.
-
-## Setup: API configuration
-This template requires app-specific configuration before it can used.
-Specifically, the template will need to be informed of the authorization
-details, and certain adjustments made to ensure the correct data can be
-extracted from the responses.
-
-In the Server.gs, APICode.gs and Auth.gs project files, there are several
-comments marked as **TODO**. To configure the template, visit each of these
-**TODO** sections and follow the directions found there.
-
-Note that the template design assumes that the template will connect to an API
-service using OAuth 2.0; however, it would be possible adapt it to connect to
-a different kind of service, such as a SQL database.
-
-## Additional information
-
-For more information, see:
-
-* [Extending Google Sheets](https://developers.google.com/apps-script/guides/sheets)
-* [Known Issues specific to Google Sheets](https://developers.google.com/apps-script/migration/sheets)
-
-Note that this template must be added to a container-bound
-script attached to a Google Sheet in order to function. Developed
-add-ons must go through a
-[publishing process](https://developers.google.com/apps-script/add-ons/publish)
-before they can be made available publicly.
+# sheets-import
```

Comment:
The title should be consistent with the other titles in the template repo (initial upper case).  "Template: Importing to Sheets" or something similar would be more consistent.